include/logging/log_core.h

```diff
@@ -407,6 +407,14 @@ static inline char z_log_minimal_level_to_char(int level)
 extern struct log_source_const_data __log_const_start[0];
 extern struct log_source_const_data __log_const_end[0];
 
+/**@brief Enum with possible actions for strdup operation.
+ */
+enum _strdup_action {
+	_strdup_skip,
+	_strdup_execute,
+	_strdup_check_if_not_already_duplicated_then_execute
```

Comment:
this is way too long. Could it be `LOG_STRDUP_CHECK_EXEC`? and comment explaining action better.